### PR TITLE
feat(tui): add Turkish (tr) locale

### PR DESCRIPTION
**Pull request by fujiapple852**

### CODE REVIEW

File: crates/trippy-tui/locales/app.yml [hunk 1]
```diff
@@ -2,330 +2,440 @@ _version: 2
...
   fr: "flux"
+  tr: "akışlar"
 target:
   en: "Target"
   fr: "Cible"
+  tr: "Hedef"
 config:
   en: "Config"
   fr: "Config"
+  tr: "Yapılandırma"
```
[the remark this answers is not in the document]
Comment (arda-guler) [reply crates/trippy-tui/locales/app.yml:49]: "Yapılandırma" could stay IMHO, but maybe konfigürasyon is more familiar to computer folks(?)

File: crates/trippy-tui/locales/app.yml [hunk 2]
```diff
@@ -2,330 +2,440 @@ _version: 2
...
   fr: "Pos"
+  tr: "Poz"
 ext:
   en: "Ext"
   fr: "Ext"
+  tr: "Uzantı"
 help_tagline:
   en: "A network diagnostic tool"
   fr: "Un outil de diagnostic réseau"
+  tr: "Bir ağ teşhis aracı"
```
[the remark this answers is not in the document]
Comment (arda-guler) [reply crates/trippy-tui/locales/app.yml:253]: Nitpick: "Bir..."

File: crates/trippy-tui/locales/app.yml [hunk 3]
```diff
@@ -2,330 +2,440 @@ _version: 2
...
   fr: "Appuyez sur [%{key}] pour afficher toutes les colonnes"
+  tr: "Tüm sütunları görmek için [%{key}] tuşuna basın"
 help_license:
   en: "Distributed under the Apache License 2.0"
   fr: "Distribué sous licence Apache 2.0"
+  tr: "Apache Lisansı 2.0 altında dağıtılmıştır"
 help_copyright:
   en: "Copyright 2022 Trippy Contributors"
   fr: "Copyright 2022 Contributeurs de Trippy"
+  tr: "Telif Hakkı 2022 Trippy Katkıda Bulunanlar"
```
[the remark this answers is not in the document]
Comment (arda-guler) [reply crates/trippy-tui/locales/app.yml:273]: Nitpick: I'm a little conflicted between Trippy'e and Trippy'ye

File: crates/trippy-tui/locales/app.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (orhun) [reply crates/trippy-tui/locales/app.yml:49]: yup

File: crates/trippy-tui/locales/app.yml (hunk 3 above)
[the remark this answers is not in the document]
Comment (orhun) [reply crates/trippy-tui/locales/app.yml:273]: both works fine

File: crates/trippy-tui/locales/app.yml [hunk 4]
```diff
@@ -2,330 +2,440 @@ _version: 2
...
   fr: "Statut"
+  tr: "Durum"
 protocol:
   en: "protocol"
   fr: "protocole"
+  tr: "protokol"
 as-info:
   en: "as-info"
   fr: "info-as"
+  tr: "as-bilgi"
```
[the remark this answers is not in the document]
Comment (fujiapple852) [reply crates/trippy-tui/locales/app.yml:61]: @orhun here `as` is referring to [Autonomous System](https://en.wikipedia.org/wiki/Autonomous_system_(Internet)), the original English text is poor.  So maybe `AS Bilgi`?

[the remark this answers is not in the document]
Comment (orhun) [reply crates/trippy-tui/locales/app.yml:61]: Ha, yeah, that makes sense then!

[the remark this answers is not in the document]
Comment (orhun) [reply crates/trippy-tui/locales/app.yml:61]: Even better "AS Bilgisi"

**Comment by fujiapple852** [comment]
All LGTM, thanks!

Here is how the header looks:

```
╭───────────────────────────────────────────────────────────────── trippy v0.12.0-dev ──────────────────────────────────────────────────────────────────╮
│Hedef: 192.168.1.21 (192.168.1.21) -> 1.1.1.1 (1.1.1.1)                                                                       2024-10-04T17:12:41+08:00│
│Konfigurasyon: protokol=icmp(v4, ayrıcalıksız) AS Bilgi=açık ayrıntılar=kapalı maksimum hostlar=otomatik, gizlilik=kapalı       hYardım sAyarlar qÇıkış│
│Durum: Çalışıyor, 9 atlanan keşfedildi                                                                                                                 │
╰───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
```

Vs `en`:

```
╭───────────────────────────────────────────────────────────────── trippy v0.12.0-dev ──────────────────────────────────────────────────────────────────╮
│Target: 192.168.1.21 (192.168.1.21) -> 1.1.1.1 (1.1.1.1)                                                                      2024-10-04T17:13:31+08:00│
│Config: protocol=icmp(v4, unprivileged) as-info=on details=off max-hosts=auto, privacy=off                                           help settings quit│
│Status: Running, discovered 9 hops                                                                                                                     │
╰───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
```

I think I need to rework the 2nd (Config) line to look better for languages which use multiple words for "as-info" and "max-hosts".  

I also need to improve the help/settings/quit part on the right of that line for non-English languages.  

But neither of these issues are specific to `tr` so don't block merging this PR.

**Pull request merged**